package.json

```
@@ -17,7 +17,7 @@
     "pretest:all": "yarn build:all",
     "test:all": "jest --coverage --passWithNoTests && lerna run test --scope @aws-sdk/stream-collector-browser --scope @aws-sdk/hash-blob-browser",
     "test:functional": "jest --config tests/functional/jest.config.js --passWithNoTests",
-    "test:integration": "cucumber-js --fail-fast",
+    "test:integration": "cucumber-js --fail-fast && jest --config jest.config.integ.js --passWithNoTests",
```

Rename existing integration tests to `test:integration-legacy` and add a new script instead.
```suggestion
    "test:integration-legacy": "cucumber-js --fail-fast",
    "test:integration": "jest --config jest.config.integ.js --passWithNoTests",
```

This way, `--tags` can still be passed while running `test:integration-legacy`
For example, `yarn test:integration-legacy --tags @s3`